apis/kueue/v1alpha1/cohort_types.go

```diff
@@ -34,11 +34,7 @@ type CohortSpec struct {
 	// Cohort, including ClusterQueues, until the cycle is
 	// removed.  We prevent further admission while the cycle
 	// exists.
-	//
-	//+kubebuilder:validation:MaxLength=253
-	//+kubebuilder:validation:Pattern="^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$"
-	//
-	Parent string `json:"parent,omitempty"`
+	Parent kueuebeta.CohortReference `json:"parent,omitempty"`
```

Comment:
Please separate introduction of CohortReference to separate PR, and use whenever easy to do so (for the ease of reviewing).

Reply:
ack, ok